src/fides/api/models/privacy_experience.py

```diff
@@ -308,62 +285,7 @@ def get_related_privacy_notices(
             )
             privacy_notice_query = privacy_notice_query.filter(PrivacyNotice.data_uses.overlap(data_uses))  # type: ignore
 
-        if not fides_user_provided_identity:
-            return privacy_notice_query.order_by(PrivacyNotice.created_at.desc()).all()
-
-        notices: List[PrivacyNotice] = []
-        for notice in privacy_notice_query.order_by(PrivacyNotice.created_at.desc()):
-            cache_saved_and_served_on_consent_record(
-                db=db,
-                consent_record=notice,
-                fides_user_provided_identity=fides_user_provided_identity,
-                record_type=ConsentRecordType.privacy_notice_id,
-            )
-            notices.append(notice)
-
-        return notices
-
-    def update_with_tcf_contents(
-        self,
-        db: Session,
-        base_tcf_contents: TCFExperienceContents,
-        fides_user_provided_identity: Optional[ProvidedIdentity],
-    ) -> bool:
```

Comment:
Method moved out of the model and adapted in Plus - this adds the TCF contents to the Privacy Experience but this move lets us also port more TCF schemas to Plus.